src/Arcus.Messaging.Pumps.ServiceBus/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -246,5 +277,19 @@ private static void AddServiceBusMessagePump<TMessagePump>(IServiceCollection se
             services.AddSingleton(serviceProvider => new AzureServiceBusMessagePumpSettings(entityName, subscriptionName, getConnectionStringFromConfigurationFunc, getConnectionStringFromSecretFunc, messagePumpOptions, serviceProvider));
             services.AddHostedService<TMessagePump>();
         }
+
+        /// <summary>
+        ///     Adds a <see cref="IAzureServiceBusMessageHandler{TMessage}"/> implementation to process the messages from Azure Service Bus resources.
```

Comment:
Shouldn't be specific for Service Bus
```suggestion
        ///     Adds a message handler to process the messages.
```

Reply:
See comment above... this message pump can only handle those kind of message/contexts. We should maybe change the signature.

Reply:
Added a different extension for that.